packages/tools/devtools/devtools-view/src/components/data-visualization/FluidValueView.tsx

```diff
@@ -31,8 +27,7 @@ export function FluidValueView(props: FluidValueViewProps): React.ReactElement {
 	const { label, node, containerKey } = props;
 	const { containerFeatureFlags } = useContainerFeaturesContext();
 	const editingEnabled =
-		containerFeatureFlags[ContainerDevtoolsFeature.ContainerDataEditing] === true &&
-		node.editProps !== undefined;
+		containerFeatureFlags.containerDataEditing === true && node.editProps !== undefined;
```

Comment:
It doesn't because we haven't enabled this for customers yet. On the devtools-core side, we have it globally disabled. You can test it locally by toggling the flag in the code, but not customers have seen it yet.